src/OrchardCore.Modules/OrchardCore.Localization/CultureSettings.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.Localization;
+
+public enum CultureSettings
```

Comment:
Could be moved to `OC.Abstraction` if we want to add it to `ILocalizationService`